web/src/ts/services/API/index.ts

```diff
@@ -0,0 +1,22 @@
+import axios from "axios"
+
+import { IUser } from './../../types/index'
+
+const API_URL = process.env.API_URL
+
+const $api = axios.create({
+  withCredentials: true,
+  baseURL: `${API_URL}/api/v1`,
+})
+
+export const userAPI = {
+  registrationUser(email: string, username: string, password: string, passwordConfirmation: string) {
```

Comment:
You will always pass each data? It is not cool, but a lot of code will be here...
  Try do smthing like:
  `registrationUser(user: UserType) return $api.post<IUser>('/users', data)`

  This - `.then(response => response.data)` - shoud move to reducer

Reply:
userCreate according to CRUD